### PR TITLE
Correct parameter order in FSM transition-syntax snippet

### PR DESCRIPTION
**Pull request by nathanmbrown**
To match example context and Scala version.

### CODE REVIEW

**Review by raboof — COMMENTED** [review]
Seems sensible to me! Apart from completing the online CLA, could you also update the javadsl example?

File: akka-docs/src/test/java/jdocs/actor/fsm/FSMDocTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void handler(StateType from, StateType to) {
 
     //#transition-syntax
     onTransition(
-      matchState(Active, Idle, () -> setTimer("timeout",
+      matchState(Idle, Active, () -> setTimer("timeout",
         Tick, Duration.ofSeconds(1L), true)).
       state(Active, null, () -> cancelTimer("timeout")).
```
Comment (patriknw) [inline akka-docs/src/test/java/jdocs/actor/fsm/FSMDocTest.java]: makes sense since the timer is cancelled when moving from Active

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by nathanmbrown** [comment]
@raboof Is there a particular file you are referring to? I can't see the same error in any other file...

**Review by raboof — APPROVED** [review]
I meant https://github.com/akka/akka/blob/4bee6aa833cc7dba13ab3c85c2d4573436047c56/akka-docs/src/test/scala/docs/actor/FSMDocSpec.scala#L118-L122, but it seems that's already OK! :+1:

**Pull request merged**